extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/runtime/SchedulerRuntimeConfig.java

```diff
@@ -13,4 +15,9 @@
     @ConfigItem(defaultValue = "true")
     public boolean enabled;
 
-}
\ No newline at end of file
+    /**
+     * Scheduled task will be flagged as overdue if next execution time is exceeded by this period.
+     */
+    @ConfigItem(name = "grace-period", defaultValue = "1S")
```

Comment:
```suggestion
    @ConfigItem(defaultValue = "1")
```

Reply:
The name is not needed since it can be generated from the variable name.

If the config is of type Duraton and it is just a number, it is automatically considered as seconds.

Reply:
it's done